#
# Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved.
#
# This patch fixes:
# Bug 16807596 - [XKB] Sun Type 7 Keyboard Scroll lock and
#    Compose indicators (LEDs) don't work
#
--- a/compat/ledcompose	1970-01-01 01:00:00.000000000 +0100
+++ b/compat/ledcompose	2013-08-21 11:51:25.761281763 +0200
@@ -0,0 +1,16 @@
+
+default partial xkb_compatibility "compose"  {
+
+    virtual_modifiers   Compose;
+
+    interpret Multi_key+Any {
+	virtualModifier= Compose;
+	action = LatchMods(modifiers=modMapMods);
+    };
+
+   indicator "Compose" {
+       allowExplicit;
+       whichModState= Latched;
+       modifiers= Compose;
+   };
+};
